src/components/Person/Person.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      <p className="Person__age">
+        {age ? `I am ${age}` : null}
+      </p>
```

Comment:
I think that you should render the whole paragraph conditionally, otherwise you might end up with empty elements, like in the screenshot. 
![image](https://user-images.githubusercontent.com/75549681/170962340-28ef3ad4-e37b-4fa1-83ac-b90a64b2aed1.png)